typescript/src/commands/Giveaway/gend.ts

```diff
@@ -0,0 +1,46 @@
+import { kRawEmoji } from '#lib/database';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import type { GuildMessage } from '#lib/types';
+import { PermissionLevels } from '#lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Message } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	aliases: ['ge', 'giveaway-end'],
```

Comment:
The alias can also be `gend` itself.

Reply:
`gend` itself is included in the command name.